pix2tex/dataset/latex2png.py

```diff
@@ -122,9 +140,32 @@ def tex2png(eq, **kwargs):
 
 
 def tex2pil(tex, **kwargs):
-    pngs = Latex(tex, **kwargs).write(return_bytes=True)
+    pngs, error_index = Latex(tex, **kwargs).write(return_bytes=True)
     images = [Image.open(io.BytesIO(d)) for d in pngs]
-    return images
+    return images, error_index
```

Comment:
Add an keyword argument that determines if `error_index` should be returned with default False. Would break in `cli` otherwise

Reply:
Added